release-calendar/src/client/components/App.tsx

```diff
@@ -13,10 +13,19 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
 import * as React from 'react';
+import {Calendar} from './Calendar';
+//TODO: remove fakeData and instead populate with data from ./test/development-data.ts and then from real data
+import {FAKEEVENTS} from '../fakeData/fakeEvents';
```

Comment:
Instead of loading data here, does it make sense to load data in the calendar component itself?
https://reactjs.org/docs/faq-ajax.html#how-can-i-make-an-ajax-call